pkgs/by-name/sk/skia/package.nix

```diff
@@ -62,7 +63,15 @@ stdenv.mkDerivation (finalAttrs: {
     vulkan-memory-allocator
   ];
 
-  configurePhase = ''
+  configurePhase = let
+    cpu =
+         ({ "x86_64" = "x64";
+            "i686" = "x86";
+            "arm" = "arm";
+            "aarch64" = "arm64";
+          }.${stdenv.hostPlatform.parsed.cpu.name}
+        or (throw "no entry for cpu: ${stdenv.hostPlatform.parsed.cpu.name}"));
```

Comment:
There's an extra pair of parentheses around the expression for `cpu`. I think the ones around `throw` are also redundant.